froide/document/templates/filingcabinet/_base_set_form.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+<div class="mb-3 d-none" id="{{ form_id }}" data-autofocus="{%block autofocus%}{%endblock%}">
+  <form method="post" class="{% block fromclasses %}disable-submit{% endblock %}" action="{%block saveurl %}{%endblock%}">
```

Comment:
We don't have a style guide for template tags, but maybe we need one! We usually use spacing like in the Django docs: `{% block name %}`

Reply:
Ooops, missed that